pyproject.toml

```diff
@@ -35,6 +30,65 @@ classifiers = [
     'Operating System :: MacOS',
 ]
 dynamic = ['version']
+dependencies = [
+    # TODO: update to "pin-compatible" once possible, see
```

Comment:
What does this mean? I have no idea.

Reply:
Beats me. I think it was just copied from scipy. I almost removed it, but wasn't sure. I am happy to just remove it.